GOVERNANCE.md

```diff
@@ -0,0 +1,164 @@
+# Open Cluster Management Project Governance
+
+* [Values](#values)
+* [Individual Subproject Governance](#individual-Subproject-governance)
+* [Steering Committee](#steering-committee)
+  * [Steering Committee Duties](#steering-committee-duties)
+  * [Steering Committee Elections](#steering-committee-elections)
+* [Code of Conduct Committee](@code-of-conduct-committee)
+* [Adding New Subprojects](#adding-new-Subprojects)
+* [Remocing Projects](#removing-projects)
+
+The Open Cluster Management umbrella project is composed as a federation of individual projects,
+some independent and some interdependent, each of which focuses on some aspect
+of Kubernetes cluster and multicluster management.  Our governance reflects this federated structure.
+
+## Values
+
+Open Cluster Management and its leadership embrace the following values:
+
+* [TODO: List of Values]
+
+<!-- 
+This is where you put the core values or principles of your project, like
+openness, distributed design, fairness, diversity, etc.
+See https://contribute.cncf.io/maintainers/governance/charter for guidance 
+and examples.
+-->
+
+## Individual Subproject Governance
+
+All active Maintainers of each Subproject, as defined in the Contributor Ladder, are
+members of that project's Maintainer Committee, which governs that project.  The
+Subproject's Maintainer Committee is responsible for the following project governance
+activities:
+
+* Ensuring that the Subproject creates and publishes regular releases;
+* Holding regular, Subproject-wide discussions on issues and planning;
+* Monthly review of project contributors for advancement on the Contributor Ladder;
+* Making final decisions on Subproject changes that involve controversial trade-offs;
+* Responding to security compromise reports;
+* Supporting the Code of Conduct within their Subproject and referring violations
+  to the Code of Conduct Committee.
+
+Additionally, the project's Maintainer Committee will select, by majority vote, one
+representative to the Steering Committee.  This representative need not
+be a member of the Subproject's Maintainer Committee, and will be replaced or
+renewed by the committee annually.
+
+Should a member of the Maintainer Committee cease being active in the Subproject,
+violate the Code Of Conduct, or need to be removed for some other reason, they
+may be removed by a 2/3 majority vote of the other Committee members, or a
+majority vote of the Steering Committee.
+
+## Steering Committee
+
+The overall Open Cluster Management umbrella project is governed by a Steering
+Committee, which is selected as follows:
+
+* Two Maintainer representatives from each member Subproject
```

Comment:
should we have a total number of Steering Committee? Currently there are 17 repos, which I think belongs to 4 sub projects.

Reply:
Oh, I thought there were only two subprojects.  In that case, does it make more sense to have one rep per subproject?

Reply:
create a PR https://github.com/open-cluster-management-io/community/pull/75 to define the subproject clearer

Reply:
The subprojects are now defined in https://github.com/open-cluster-management-io/ocm

- [Foundation / Cluster Lifecycle](https://github.com/open-cluster-management-io/ocm#cluster-lifecycle-cluster-registration-and-management)
- [Application Lifecycle](https://github.com/open-cluster-management-io/ocm#application-lifecycle-delivery-upgrade-and-configuration-of-applications-on-kubernetes-clusters)
- [GRC: Governance, Risk and Compliance across Kubernetes clusters](https://github.com/open-cluster-management-io/ocm#grc-governance-risk-and-compliance-across-kubernetes-clusters)
- [Additional External Integrations](https://github.com/open-cluster-management-io/ocm#more-external-integrations)

CC @juliana-hsu